src/nuclio/functions/function-collapsing-row/function-version-row/function-version-row.component.js

```diff
@@ -131,6 +134,18 @@
             ctrl.toggleFunctionState({event: event})
         }
 
+        //
+        // Public methods
+        //
```

Comment:
why did you create a new section comment `Public methods`. We already have it in this component.